### DISCUSSION TITLE
What does check_model() do for mixed beta regression?

### DISCUSSION THREAD

**Discussion by tbx200** [easystats/performance#357] [Q&A] [answered]
I'm fitting a mixed beta-regression model in glmmTMB. 
After reading some literature (Espinheira et al. 2008 & Ferrari & Cribari-Neto 2004), I'm wondering what exactly is calculated and plotted for this type of model. I understand that model diagnostics for this type of model have not been fully crystalized yet.

Are the calculated residuals, weighted residuals (and which type), or are they standardized ordinary residuals?

**Comment by bwiernik** [comment, marked as answer]
It uses the standard `residuals()` function from the *glmmTMB* package. The default for this function is `"response"`, that is, simply `res = response - fitted` residuals, not weights or scaling. There is potentially an argument that we should use weighted Pearson residuals instead, but currently the default response residuals are used.
